File: src/main/resources/template/ios/IosLauncher.java
package ${package};

import com.jme3.system.ios.IosHarness;

class IosLauncher extends IosHarness{
    public IosLauncher(long appDelegate) {
        super(appDelegate);
    }

    @Override
    public void appPaused() {

    }

    @Override
    public void appReactivated() {
      app=new Main();
      app.start();
    }

    @Override
    public void appClosed() {
        app.stop();
    }

    @Override
    public void appUpdate() {

    }

    @Override
    public void appDraw() {
    }

    @Override
    public void appReshape(int width, int height) {

    }
}

File: src/main/resources/template/ios/Ioslauncher.kt
package ${package}

import com.jme3.system.ios.IosHarness

internal class IosLauncher(appDelegate: Long) : IosHarness(appDelegate) {

    override fun appPaused() {

    }

    override fun appReactivated() {
        app = Main()
        app.start()
    }

    override fun appClosed() {
        app.stop()
    }

    override fun appUpdate() {

    }

    override fun appDraw() {}

    override fun appReshape(width: Int, height: Int) {

    }
}
